scripts/clangfmt

```diff
@@ -61,14 +61,31 @@ while [ "$#" -gt 0 ]; do
   esac
 done
 
+# Use this block for version 10 and above
+# if [ "$test_mode" = true ]; then
+#   opts="--dry-run"
+#   err="--Werror"
+# else
+#   opts="-i"
+#   err=
+# fi
+
+# Remove for version 10 and above
+opts="-i"
+err=
+
 if [ "$#" -gt 0 ]; then
-  "$clang_format" --style=file -i "$@"
+  "$clang_format" --style=file "$opts" "$@"
 else
-  find . -name "*.[ch]" -or -name "*.cpp" | xargs "$clang_format" --style=file -i
+  find . -name "*.[ch]" -or -name "*.cpp" | \
```

Comment:
What about `.hpp` extensions? I think some people are also using them. We even have few of such files in `libunwind`.

Reply:
Yes, this has been added now.